docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
 ---
 title: "User administration"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a set of permissions. Depending on your role, you'll be able to access different levels of the application, environments and projects. If you want to add a user to a project or an environment type, the user has to create an account before they can contribute to the project.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+### User roles
 
-## User roles
```

Comment:
Why this change? It's the first heading, so it should be an h2. If there's no h2 and it skips right to h3, the document isn't semantically valid.